dist/main.css

```diff
@@ -0,0 +1,11 @@
+svg {
+  width:"2338";
+  height:"1080";
```

Comment:
Consider using a percent instead of a hard coded value.

Also, with SVG 2 it's best practice to specify units. It defaults to pixels but better to be specific. If you want more control than percent you can also set it as `em`, use `em` instead of `px`. em is based on font size, it gives the user some control over size by using `<command <+>` or `<control <+>`